src/gui/guiFuncs.cpp

```diff
@@ -401,18 +407,23 @@ void DissolveWindow::fullUpdate()
 // Update while running
 void DissolveWindow::updateWhileRunning(int iterationsRemaining)
 {
+
     refreshing_ = true;
 
     // Set current iteration number
     iterationLabel_->setText(QStringLiteral("%1").arg(dissolve_.iteration(), 6, 10, QLatin1Char('0')));
 
+    // Text is set to time elapsed if iterating indefinitely
+    if (iterationsRemaining == -1)
+    {
+        timerLabel_->setText(QString::fromStdString(elapsedTimer_.elapsedTimeString(true)));
+    }
```

Comment:
For one-liners after an `if` I tend to omit the braces. Why take up three lines when you only need one!
```suggestion
    if (iterationsRemaining == -1)
        timerLabel_->setText(QString::fromStdString(elapsedTimer_.elapsedTimeString(true)));
```